scripts/encapp.py

```diff
@@ -780,9 +819,9 @@ def main(argv):
         if input_filepath != "camera" and not encapp_tool.ffutils.video_is_raw(
             input_filepath
         ):
-            # TODO(chema): process compressed video
-            # videofile_config = get_video_info(input_filepath)
-            pass
+            # replace input and other derived values
```

Comment:
This must be optional (i.e. transcode the input if an encoded file).

Reply:
It is, right? 

The code above is:
```
        if input_filepath != "camera" and not encapp_tool.ffutils.video_is_raw(
            input_filepath
        ):
            # replace input and other derived values
            d = process_input_path(input_filepath, options.replace, options.debug)      
            options.replace["input"].update(d)
```

And `encapp_tool.ffutils.video_is_raw()` checks for an extension in:

```
RAW_EXTENSION_LIST = (".yuv", ".rgb", ".rgba", ".raw")
```